core/src/main/java/io/crate/monitor/ExtendedFsStats.java

```diff
@@ -47,6 +54,12 @@
         private long diskReadBytes = -1;
         private long diskWriteBytes = -1;
 
+        public static Info readInfo(StreamInput in) throws IOException {
```

Comment:
Maybe you could use `fromStream` as the standard name for this builder method (as you have on `Version` and `Build`). Also apply it to the other classes that implements `Streamable`.